packages/standard-minifier-js/package.js

```diff
@@ -1,23 +1,20 @@
 Package.describe({
   name: 'standard-minifier-js',
-  version: '1.2.3',
+  version: '2.0.0',
   summary: 'Standard javascript minifiers used with Meteor apps by default.',
-  documentation: 'README.md'
+  documentation: 'README.md',
 });
 
 Package.registerBuildPlugin({
   name: "minifyStdJS",
   use: [
-    'minifier-js'
+    'minifier-js@2.0.0',
```

Comment:
After talking about this today, we're going to leave the constraint here for now, since it needs to be there when we publish the package, and shouldn't be hard to remember to update in the future (unlike some other package constraints, which would appear in too many different `package.js` files to keep track of).